src/Files.Uwp/ViewModels/ToolbarViewModel.cs

```diff
@@ -329,7 +329,7 @@ public CurrentInstanceViewModel InstanceViewModel
 
         private PointerRoutedEventArgs pointerRoutedEventArgs;
 
-        public NavToolbarViewModel()
+        public ToolbarViewModel()
```

Comment:
Should this be `AddressToolbarViewModel`?

Reply:
The same ViewModel is used by the InnerNavigationToolbar as well. I thought a common name would be best, though I'm happy to change it?

Reply:
I'm not familiar with what it's used for off hand and it may be beyond the scope of this PR but should each control have it's own view model attatched?

Reply:
@yaichenbaum I think that would be out of scope for this PR. I think the question is whether they share the same ViewModel instance or if @puppetsw just meant they are separate instances with the same, generic name.

Reply:
@lukeblevins I believe they are both using the same instance of that ViewModel.